<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Amir</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Port Status</string>
    </property>
    <widget class="QLabel" name="port_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Port Status!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close Port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Received</string>
    </property>
    <widget class="QLineEdit" name="ByteR_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ByteR_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bytes Received:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Text Files</string>
    </property>
    <widget class="QLineEdit" name="python_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>18</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>56</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="plot_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Python Output:</string>
     </property>
    </widget>
    <widget class="QLabel" name="plot_label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>C++ Output:</string>
     </property>
    </widget>
    <widget class="QLabel" name="output_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>94</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Output Status!</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="plot_label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>RS232 PROTOCOL (PYTHON Programming Language)</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>plot_label_4</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="file_actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="file_actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
